bin/_info

```diff
@@ -89,6 +89,16 @@ def main() -> int:
     print()
     with _console():
         _call('ruby', '--version')
+    print()
+
+    print('## rust')
+    print()
+    with _console():
+        _call('rustup', '--version')
+        print()
```

Comment:
I don't think they'll be able to run `rustup` so just cargo and rustc are relevant here? (though admittedly I'm not super knowledgeable in this space so I could be wrong here)

Reply:
I'm not sure what do you mean here. One possible use of rustup that comes to mind is if somebody used this as a base image and wants to install another toolchain.

Reply:
that's not a use case I intend to support so it's not worth documenting